### DISCUSSION TITLE
Has anyone seen this error before? Kustomization reconciliation failed: too many fields within:

### DISCUSSION THREAD

**Discussion by LeoViniciusCS** [fluxcd/flux2#1989] [Q&A]
Good afternoon everyone!

I just upgraded to Flux v0.19.0 and some kustomizations that were working before  start giving me this error:
    `Kustomization reconciliation failed: too many fields within: lh-staff_developer_rbac.authorization.k8s.io`

Is there any way to get an explanation or a more verbose output on this error?

Another weird thing I just noticed is, this resource doesn't exist:
    `lh-staff_developer_rbac.authorization.k8s.io` 
The name in the error should be:
    `lh-staff_developer.rbac.authorization.k8s.io`

Thanks in advance for all your time and attention :)

Regards

**Comment by stefanprodan** [comment]
You could replicate the error locally with `kubectl apply --server-side -f`, maybe kubectl says more?

**Reply by LeoViniciusCS** [reply]
Hi! Thanks for the quick reply!

I tried to replicate it locally, but the error doesn't happen:
`$    kubectl apply --server-side -f rb_developer.yml -n lh-common-dev`
`rolebinding.rbac.authorization.k8s.io/lh-staff_developer serverside-applied`

**Reply by stefanprodan** [reply]
Can you please open an issue in kustomize-controller repo and provide an RBAC YAML so we can replicate it?

**Reply by castorls** [reply]
I've the same issue. 
Issue created in kustomize-controller project : https://github.com/fluxcd/kustomize-controller/issues/491

downgrade kustomize-controller to 0.14.1 fix the issue for me.
